arangodb-foxx-services/src/main/utils/logger.ts

```diff
@@ -19,6 +19,10 @@ import { context } from '@arangodb/locals'
 
 const isDebugEnabled: boolean = context.isDevelopment
```

Comment:
@wajda I would recommend using standard packages for logging https://npmcompare.com/compare/bunyan,loglevel,npmlog,pino,winston

Reply:
I wonder if anything of those can be used in Foxx, which I doubt, but I'll take a look. Thanks for the hint.

Reply:
Glanced through all the libraries and... No.
They all are designed for a pure Nodejs or Browser logging, supporting additional levels, fancy formatting, alternative streams, log rotations etc etc etc. But nothing of those feature are required in this app. All logging in Foxx is done via `Console`, that are then routed to the internal ArangoDB logging system which then does all logging management centrally. Unfortunately there is Foxx specific appender for any of those frameworks (or at all), so we'd have to write our own appender anyway. All of that above tells me that using any external logging library for this app is simply unnecessary over engineering.